src/bin/ion/commands/inspect.rs

```diff
@@ -212,11 +212,11 @@ impl<'a, 'b> IonInspector<'a, 'b> {
         loop {
             let item = reader.next_item()?;
             let is_last_item = matches!(item, SystemStreamItem::EndOfStream(_));
+            item.raw_stream_item()
+                .map(|i| self.confirm_encoding_is_supported(i.encoding()))
+                .unwrap_or(Ok(()))?;
```

Comment:
🗺️ I revised this section to fix a layout bug that would cause the table header to be output multiple times if you were using `--skip-bytes`.